### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Ansagan-B**

### CODE REVIEW

**Comment by qwaseras** [comment]
You have 3 commits with the same name. Better describe what you did and what features you implemented (or fixed)

File: workflow/landing-page/index.html [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+        <div class="icon">
+            <a class="fa fa-rss" aria-hidden="true" href="#"></a>
+        </div>
+    </div>
+</header>
+
+<div class="navbar">
+    <div class="icons" id="nav-icon">
+        <div class="nav-button icon">
+            <div id="myNav" class="overlay closedNav">
```
Comment (qwaseras) [inline workflow/landing-page/index.html:35]: dont call something like myNav

File: workflow/landing-page/index.html [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```
Comment (qwaseras) [inline workflow/landing-page/index.html:36]: you can leave href empty, javascript:void(0) means nothing in this case

File: workflow/landing-page/index.html [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+        <div class="box">
+            <div class="box-text">
+                <h4>Creative Idea Important</h4>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                    Consequuntur, odit, magnam, autem ipsum incidunt modi
+                    saepe id ullam consectetur unde minima quis.
+                </p>
+            </div>
+            <div class="animation"><i class="anmtn fa fa-lightbulb-o" aria-hidden="true"></i>
```
Comment (qwaseras) [inline workflow/landing-page/index.html:134]: also, dont use shortcuts like anmtn - it's hard to understand what it means
for example, you could call it "animation-icon"

File: workflow/landing-page/index.html [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+        <form id="form">
+            <p id="contactPage">
+                This is an example of a contact page. You can set the contact page
+                in your themes <br>backend and add, remove and modify the input fields, text areas,
+                dropdowns and <br>checkboxes from your backend as well.
+            </p>
+            <h4>
+                Send Us Email
+            </h4>
+            <p>
```
Comment (qwaseras) [inline workflow/landing-page/index.html:346]: Don't use _p_ tag to group the elements, it is just for text
use div or something like this

File: workflow/landing-page/index.html (hunk 1 above)
Comment (qwaseras) [reply workflow/landing-page/index.html:35]: also - better use kebab-case, like "long-id-kebab-name" (not camelCase - longCamelCaseName)

Comment (Ansagan-B) [reply workflow/landing-page/index.html]: Thanks Roman,

Refactored, all `camelCases` to `kebab-case`.

File: workflow/landing-page/index.html (hunk 4 above)
Comment (Ansagan-B) [reply workflow/landing-page/index.html]: Changed `p` tag to `div` to group elements.

**Review by qwaseras — APPROVED** [review]

**Pull request merged**